chain/westend-dev/defaults.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package westenddev
+
+import (
+	"time"
+
+	cfg "github.com/ChainSafe/gossamer/config"
+
+	"github.com/ChainSafe/gossamer/dot/state/pruner"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
+)
+
+const (
+	// DefaultName is the default node name
+	DefaultName = "Westend"
+	// DefaultID is the default node ID
+	DefaultID = "westend_dev"
+	// DefaultBasePath is the default basepath for the westend dev node
+	DefaultBasePath = "~/.gossamer/westend-dev"
+	// DefaultChainSpec is the default chain spec for the westend dev node
+	DefaultChainSpec = "./chain/westend-dev/westend-dev-spec-raw.json"
+	// DefaultLogLevel is the default log level for the westend dev node
+	DefaultLogLevel = "info"
+	// DefaultPrometheusPort is the default metrics port for the westend dev node
+	DefaultPrometheusPort = ":9876"
+	// DefaultRetainBlocks is the default number of blocks to retain
+	DefaultRetainBlocks = uint32(512)
+	// DefaultPruning is the default pruning mode
+	DefaultPruning = pruner.Archive
+
+	// DefaultRole is the default node role
+	DefaultRole = common.AuthorityRole
+	// DefaultWasmInterpreter is the default wasm interpreter
+	DefaultWasmInterpreter = wasmer.Name
+	// DefaultBabeAuthority is the default babe authority
+	DefaultBabeAuthority = true
+	// DefaultGrandpaAuthority is the default grandpa authority
+	DefaultGrandpaAuthority = true
+	// DefaultGrandpaInterval is the default grandpa interval
+	DefaultGrandpaInterval = time.Second
```

Comment:
Is one second the actual default grandpa interval on westend? Why don't we use westend defaults for the local and dev networks?

Reply:
To clarify, I'm not necessarily saying we are wrong not to, mostly just trying to understand myself

Reply:
If this is left as zero value in the config, it gets updated in `lib/grandpa/grandpa.go#131`

Reply:
The default value for that is `1s`!